src/style.css

```diff
@@ -0,0 +1,47 @@
+
+* {
+  padding: 0;
+  box-sizing: border-box;
+  margin: 0;
+}
```

Comment:
it is not OK to reset paddings and margins for ALL elements (`*` selector causes it), place these two rules under some other selector